### PR TITLE
image orientation while loading bitmap issue fixed.

### PR DESCRIPTION
**Pull request by mostafaimran**
There was an image orientation issue while loading bitmap into image editor. It's solved now.

Issue - https://github.com/iamutkarshtiwari/Ananas/issues/82

### CODE REVIEW

File: ananas/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<resources>
+    <string name="iamutkarshtiwari_github_io_ananas_library_name">Photo Editor</string>
```
Comment (iamutkarshtiwari) [inline ananas/src/main/res/values-fr/strings.xml:2]: This doesn't seem like a french locale. Could you please revert this?

**Review by iamutkarshtiwari — CHANGES_REQUESTED** [review]
Thank you for your contribution!

File: ananas/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (mostafaimran) [reply ananas/src/main/res/values-fr/strings.xml]: French translation added with Arabic translation support.

**Review by iamutkarshtiwari — APPROVED** [review]
Thank you for your great work!
LGTM

**Pull request merged**